content/guides/references/migration-guide.md

```diff
@@ -528,7 +528,7 @@ required for component testing
 {
   "experimentalComponentTesting": true,
   "componentFolder": "src",
-  "testFiles": "**/*spec.{js,jsx,ts,tsx}"
+  "testFiles": "**/*.cy.{js,jsx,ts,tsx}"
```

Comment:
I don't think we would want to update the filenames in the migration guide for previous versions of Cypress